subscription.go

```diff
@@ -617,6 +620,9 @@ func (sc *SubscriptionClient) Run() error {
 				}
 
 				sub := sc.context.GetSubscription(message.ID)
+				if sub == nil {
```

Comment:
shouldn't we skip `OnMessage` callback and continue the loop?

Reply:
I considered this, but it was proven faulty logic. 

Empty subscription is expected for the `connection_ack` message. If we continue the loop at that point, the initial connection_ack is skipped as well and maybe other messages too. Instead the scope of this fix it to skip `data` messages that are not mapped to a subscription, because they require a non-nil handler.

Reply:
Make sense. Anyway LGTM